<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupDialog</class>
 <widget class="QDialog" name="SignupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>587</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/illust.jpg);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>243</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="idEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;
background-image: url(:/background.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="stu">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>I'm a Student</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tea">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>I'm a Teacher</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>243</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;
background-image: url(:/background.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>95</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ok_cancel">
    <item>
     <widget class="QPushButton" name="ok_Button">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_Button">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
